diff --git a/node_modules/react-native-text-input-mask/android/build.gradle b/node_modules/react-native-text-input-mask/android/build.gradle
index c94f33f..c045381 100644
--- a/node_modules/react-native-text-input-mask/android/build.gradle
+++ b/node_modules/react-native-text-input-mask/android/build.gradle
@@ -27,7 +27,7 @@ android {
 
 buildscript {
     ext {
-        kotlinVersion = '1.4.31'
+        kotlinVersion = '1.6.10'
     }
     repositories {
         jcenter()
